tests/indent/c/if_else.c

```diff
@@ -0,0 +1,30 @@
+int foo(int x){
+    if (x > 10)
```

Comment:
do we have enough tests for if-compound statement and stronger nestings?

Reply:
I don't think so, but it is unnecessary because even before this PR, in the case of if-compound statements, the if statements were never indented to begin with, it was handled by the compound statement

Reply:
> do we have enough tests for if-compound_statement+different combinations and stronger nestings (ifs in ifs)

Tested locally and did not seem to have issues, I will push some extra tests to check coverage for it

Reply:
Ooops, miss one case

Reply:
Tests added